manifest.json

```diff
@@ -142,8 +142,8 @@
       "id": "discord-presence"
     },
     {
-      "description": "Provides basic drag and drop of selected text (in same document)",
-      "version": "0.2",
+      "description": "Provides drag and drop of selected text (in same document)",
+      "version": "20230616.094245",
```

Comment:
@adamharrison could you confirm that this is a valid version for lpm?

Reply:
yes it does:
https://github.com/lite-xl/lite-xl-plugin-manager/blob/master/SPEC.md#:~:text=version%3A%20The%20addon%27s%20semantic%20version.%20A%20string%20that%20can%20contains%20%5B0%2D9%5C.%5D.

Reply:
I mean with an `lpm` - the software - perspective. Iirc it's slightly limited in that regards.

Reply:
spec should state that if there are limits other than characters used.

I have asked: https://github.com/lite-xl/lite-xl-plugin-manager/issues/22

Reply:
It will work, but isn't convention of major.minor.revision.

Reply:
neither this nor most plugins fit into a major.minor.patch scenario :D

Reply:
minor and patch are optional; they're assumed to be 0 if omitted.